### ISSUE TITLE
distinst doesn't support elementary OS EFI path

### ISSUE THREAD

**Issue by codygarver** [elementary/os#200]
There will probably need to be some logic to map it to upstream Ubuntu's like I do in grub but I will figure this out after I figuring out the best way to implement fix for #196

**Comment by cassidyjames** [comment]
@codygarver also let @mmstick know if there's anything that can be done in distinst itself to help!

**Comment by mmstick** [comment]
distinst is actually compatible with Ubuntu ISOs and Grub. We build Ubuntu images in Trubble with distinst.

**Comment by codygarver** [comment]
@mmstick the problem isn't compatibility with GRUB, it's that distinst is trying to use elementary lsb info to guess paths that don't exist on the filesystem because of things I had to do to re-use Ubuntu's signing key.

I will ping you again when I have more info to share, I did not want to ping you until I had more but someone else on this bug did for me. I have 2 EFI related bugs I'm working on and I have to solve the one that happens earlier before I get to this one, which is due to absence of ubiquity in our images now.
